.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,40 @@
-#### What does this PR do?
-#### Any background context you want to provide?
 #### What ticket does this PR close?
 Connected to [relevant GitHub issues, eg #76]
-#### Where should the reviewer start?
-#### How should this be manually tested?
-#### Screenshots (if appropriate)
-#### Has the Version and Changelog been updated?
-#### Questions:
-> Does this work have automated integration and unit tests?
 
-> Can we make a blog post, video, or animated GIF of this?
+## Checklist
```

Comment:
I would suggest we have all of the boxes checked at first, and the checklist instructions should ask the PR submitter to read through the list and uncheck any that don't apply. I agree with @rafis3 below that the vast majority of the time, all of these will be checked (and in practice I doubt if people will actually go through and check > 10 boxes)